admin/tabs/wizard.php

```diff
@@ -4,6 +4,19 @@ function is_json_object( $array ) {
     json_encode( $array );
     return json_last_error() === JSON_ERROR_NONE;
 }
+function is_plugin_installed( $slug ) {
+    $pathpluginurl = WP_PLUGIN_DIR . '/' . $slug;
+    $is_installed = file_exists( $pathpluginurl );
```

Comment:
Instead of manually checking if the file exists, you can use the WordPress `get_plugins()` method: https://developer.wordpress.org/reference/functions/get_plugins/

I always try to avoid interacting with the file system directly if possible. It can sometimes be troublesome in different hosting environments and prone to errors if there are some sort of permission issues or something.